docs/examples.md

```diff
@@ -0,0 +1,321 @@
+Epoch 2/15
+12/12 [==============================] - 0s 3ms/step - loss: 0.6944
+Epoch 3/15
+12/12 [==============================] - 0s 3ms/step - loss: 0.7057
+Epoch 4/15
+12/12 [==============================] - 0s 3ms/step - loss: 0.6963
+Epoch 5/15
+12/12 [==============================] - 0s 3ms/step - loss: 0.6937
+Epoch 6/15
+12/12 [==============================] - 0s 3ms/step - loss: 0.6931
+Epoch 7/15
+12/12 [==============================] - 0s 3ms/step - loss: 0.6909
+Epoch 8/15
+12/12 [==============================] - 0s 3ms/step - loss: 0.6891
+Epoch 9/15
+12/12 [==============================] - 0s 3ms/step - loss: 0.6931
+Epoch 10/15
+12/12 [==============================] - 0s 3ms/step - loss: 0.6960
+Epoch 11/15
+12/12 [==============================] - 0s 5ms/step - loss: 0.6851
+Epoch 12/15
+12/12 [==============================] - 0s 3ms/step - loss: 0.6787
+Epoch 13/15
+12/12 [==============================] - 0s 3ms/step - loss: 0.6704
+Epoch 14/15
+12/12 [==============================] - 0s 3ms/step - loss: 0.6470
+Epoch 15/15
+12/12 [==============================] - 0s 3ms/step - loss: 0.6076
+```
+`Plot the learning curve!`
+
+```python
+tfp.plot_learning_curve();
+```
+![](images/tfplayground_learning_curve.jpg)
+
+``
+`Plot the predictions`
+
+```python
+fig, axn = plt.subplots(1, 2, figsize=(12, 4))
+
+for ax, subset in zip(axn, ['train', 'test']):
+    cp = tfp.plot_decision_function(ax=ax)
+    tfp.data.plot(subset=subset, ax=cp.axes)
+    
+    ax.set_title(subset, fontdict=dict(weight='bold', size=20))
+    
+
+fig.colorbar(cp);
+```
+![](images/tfplayground.jpg)
+
+`Analyze the peformance`
+
+```python
+tfp.plot_confusion_matrix();
+```
+![](images/tfplayground_confusion_matrix.jpg)
+
+
+### `Fashion MNIST`
+
+```python
+from tensorflow.keras.datasets import fashion_mnist
+
+# The data has already been sorted into training and testing sets
+(train_images, train_labels), (test_images, test_labels) = fashion_mnist.load_data()
+
+# Name of the classes
+class_names =  ['T-shirt/top', 'Trouser', 'Pullover', 'Dress', 'Coat',
+               'Sandal', 'Shirt', 'Sneaker', 'Bag', 'Ankle boot']
+
+n_classes = len(class_names)
+```
+
+
+
+Custom `ImageDataset`
+
+```python
+from src.image import ImageDataset
+
+imgds = ImageDataset((train_images, train_labels), (test_images, test_labels), class_names)
+```
+
+Plot the label counts
+```python
+imgds.plot_labelcounts();
+```
+![](images/fashionmnist_labelanalyzer_plot.JPG)
+
+View random images
+```python
+imgds.view_random_images(class_names='all', n_each=2, subset='train');
+```
+![](images/fashionmnist_imgds_rand_images.JPG)
+
+Model Performance Comparison
+```python
+from src.evaluate import ClassificationPerformanceComparer
+
+clf_comp = ClassificationPeformanceComparer(models, test_data)
+clf_comp.calculate_metric_comparison_df()
+clf_comp.plot_metric_comparison_df();
+```
+![](images/fashionmnist_model_comparison.jpg)
```

Comment:
This image doesn't show up in the github markdown render. Try adding an extra line before inserting the image. See [here](https://stackoverflow.com/questions/11915826/image-not-showing-up-in-readme-md-on-github)

Reply:
resolved